tools/build-fs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+
+# TODO: add a way to specify a different sigstruct directory
+
+# Net can be main/test/local
+if [ $# -eq 0 ]; then
+NET="test"
+else
+NET=$1
+fi
+
+if [ $NET == "local" ]; then
+echo "Local networks are not currently supported with this script"
+exit 1
+elif [ $NET == "test" ]; then
+echo "Building Full Service with testnet settings"
+NAMESPACE="test"
+export SGX_MODE=SW
+export IAS_MODE=DEV
```

Comment:
I missed this at a previous review, but testnet uses HW/PROD like mainnet.
Local network uses SW/DEV